processing/src/main/java/io/druid/segment/DimensionHandlerUtils.java

```diff
@@ -327,18 +328,18 @@ public static Long getExactLongFromDecimalString(String decimalStr)
     }
   }
 
-  public static Double nullToZero(@Nullable Double number)
+  public static Number nullToZeroDouble(@Nullable Number number)
```

Comment:
Why changing types and method name? Same below

Reply:
reverted.